docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md

```diff
@@ -51,19 +57,19 @@ layout: default.html
 Some **bold** content.
 ```
 
-Now `cd` to `myjekyllsite` and build the site:
+Now `cd` to `myjekyllsite` and serve the site with the built-in server:
 
 ```
 jekyll serve
 ```
 
-When you build the site, you get a preview URL such as `http://127.0.0.1:4001/`. The site's files are built in the `_site` folder.
+When you serve the site, you get a preview URL such as `http://127.0.0.1:4001/` (which is the same as `http://localhost:4001/`). The site's files are built into the `_site` folder by default.
 
-This is a Jekyll site at the most basic level. Here's what is happening:
+This is a Jekyll site at the most basic functional level. Here's what is happening:
 
 * The `_config.yml` file contains settings that Jekyll uses as it processes your site. An empty config file will use default values for building a Jekyll site. For example, to convert Markdown to HTML, Jekyll will automatically use the [kramdown Markdown filter](https://rubygems.org/gems/kramdown/), without any need to specify it.
 * Jekyll looks for files with [front matter tags]({% link _docs/frontmatter.md %}) (the two sets of dashed lines `---` like those in `index.md`) and processes the files (populating site variables, rendering any [Liquid](https://shopify.github.io/liquid/), and converting Markdown to HTML).
-* Jekyll pushes the content from all pages and posts into the `{% raw %}{{ content }}{% endraw %}` tags in the layout specified (`default`) in the front matter tags.
+* Jekyll pushes the content from all pages and posts into the `{% raw %}{{ content }}{% endraw %}` tag in the layout specified (`default`) in the front matter tags.
```

Comment:
True. This prompts me to carve out a separate section:
```
## 2. Liquid Structure

The 3 basic pillars defining Liquid language are:
  * tags -- (say what they are and what they do in one-line)
  * filters -- (say what they are and what they do in one-line)
  * variables -- (say what they are and what they do in one-line)
```
Whatsay? Or is there an appropriate page already out there that I can simply link to?